modules/tree/main/index.coffee

```diff
@@ -176,9 +185,7 @@ class Tree
       @selection.classed('hx-opened', false)
       @selection.selectAll('.hx-tree-node-children').style('display', 'none')
 
-    animate = if animate? then animate
-    else @options.animate
-
+    animate = if animate? then animate else @options.animate
```

Comment:
Not sure why these were on multiple lines - It didn't really help with readability so I undid it